lib/modules/dosomething/dosomething_api/plugins/restful/campaigns/1.0/DoSomethingCampaignsResource.class.php

```diff
@@ -24,10 +24,6 @@ public function publicFieldsInfo() {
       'property' => 'field_call_to_action',
     ];
 
-    $public_fields['on_mobile_app'] = [
-      'property' => 'field_on_mobile_app',
-    ];
-
```

Comment:
We should actually just kill all the Restful Module stuff. We don't actually use this anymore.